client/src/extension.ts

```diff
@@ -40,7 +40,7 @@ export async function activate(context: ExtensionContext) {
   warnOnOtherSolidityExtensions(extensionState);
 
   return {
-    isReady: () => extensionState?.indexingFinished,
+    isReady: () => !!extensionState?.client?.initializeResult,
```

Comment:
Do we know if this is necessary?

I can't find a usage in our codebase, was it added for tests then became unhooked? Or do we think that node language server needs it?

Reply:
This is for the e2e tests, to know when the LSP finished initialization. Probably there are better ways of doing it but for now it's doing the job

Reply:
Cool. That makes sense.